src/aerialmap_display.cpp

```diff
@@ -166,12 +161,6 @@ void AerialMapDisplay::updateAlpha()
   ROS_INFO("Changing alpha to %f", alpha_);
 }
 
-void AerialMapDisplay::updateFrame()
-{
-  ROS_INFO_STREAM("Changing robot frame to " << frame_property_->getFrameStd());
-  transformAerialMap();
-}
-
 void AerialMapDisplay::updateDrawUnder()
 {
   // @todo figure out why this property only applies to some objects
```

Comment:
@schra any idea what this comment meant?

Reply:
I do not know what Gareth meant by that.